gallery/src/main.scss

```diff
@@ -110,3 +110,28 @@
     display: none;
   }
 }
+
+// Accessibility Checklist Page
+.nx-list.nx-list--no-bullets {
```

Comment:
This `no-bullets` modifier seems like it should be a gallery class not an `nx-` class

Reply:
> This `no-bullets` modifier seems like it should be a gallery class not an `nx-` class

https://github.com/sonatype/sonatype-react-shared-components/pull/657/commits/430bacf58f9d7ae45a287f292edf720ba44db56d and https://github.com/sonatype/sonatype-react-shared-components/pull/657/commits/d5e3865a0eecfe3d0e95efc3c55868d5afc1c35e